admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -64,6 +60,28 @@ You can verify if the cron job has been added and scheduled by executing::
 
 Please refer to `the crontab man page`_ for the exact command syntax.
 
+Webcron
+~~~~~~~
+
+By registering your ownCloud ``cron.php`` script address as an external webcron service (for example, easyCron_), you ensure that background jobs are executed regularly. 
+To use this type of service, your external webcron service must be able to access your ownCloud server using the Internet. 
+For example::
+
+  URL to call: http[s]://<domain-of-your-server>/owncloud/cron.php
+
+AJAX
+~~~~
+
+The AJAX scheduling method is the default option. 
+However, it is also the *least* reliable. 
+Each time a user visits the ownCloud page, a single background job is executed. 
+The advantage of this mechanism, however, is that it does not require access to the system nor registration with a third party service. 
+The disadvantage of this mechanism, when compared to the `Webcron`_ service, is that it requires regular visits to the page for it to be triggered.
+
+.. note:: Especially when using the Activity App or external storages, where new
+   files are added, updated, or deleted one of the two methods below should be
```

Comment:
Thanks @phil-davis. I've made a follow-up commit to correct this oversight. Always great to know you're there, keeping me honest.